### PR TITLE
qemu: update to 8.2.0

### PR DESCRIPTION
**Pull request by vooon**
Maintainer: @yousong
Compile tested: x86_64, OpenWrt master (https://github.com/openwrt/openwrt/commit/713dc12960dd6146706a364905904b58d4d586d4)
Run tested: x86_64 host, KVM for qemu-x86_64-softmmu + Bios VM.

Description:
- Version updated to 8.2.0

### CODE REVIEW

File: utils/qemu/Makefile [hunk 1]
```diff
@@ -378,6 +377,7 @@ CONFIGURE_ARGS +=			\
 #
 CONFIGURE_ARGS +=			\
 	--audio-drv-list=''		\
+	--disable-af-xdp                \
```
Comment (BKPepe) [inline utils/qemu/Makefile:380]: This is not said in the commit.

File: utils/qemu/Makefile [hunk 2]
```diff
@@ -401,6 +401,7 @@ CONFIGURE_ARGS +=			\
 	--disable-libpmem		\
 	--disable-libssh		\
 	--$(if $(CONFIG_QEMU_DEV_USB),enable,disable)-libusb		\
+	--disable-libkeyutils           \
```
Comment (BKPepe) [inline utils/qemu/Makefile:404]: dtto

File: utils/qemu/Makefile [hunk 3]
```diff
@@ -417,6 +418,7 @@ CONFIGURE_ARGS +=			\
 	--disable-qom-cast-debug	\
 	--disable-rbd			\
 	--disable-rdma			\
+	--disable-rutabaga-gfx          \
```
Comment (BKPepe) [inline utils/qemu/Makefile:421]: dtto

File: utils/qemu/Makefile (hunk 1 above)
Comment (vooon) [reply utils/qemu/Makefile:380]: By defaul most of the fatures if disable in openwrt, so i decided to to enable feature we didn't use previosly.

File: utils/qemu/Makefile (hunk 2 above)
Comment (vooon) [reply utils/qemu/Makefile:404]: Same as for new nic.

File: utils/qemu/Makefile (hunk 3 above)
Comment (vooon) [reply utils/qemu/Makefile:421]: And again, we haven't use new graphics by default. Also as i know that more useful for the cloud, than wrt like single node appilcations.

Comment (vooon) [reply utils/qemu/Makefile:421]: @BKPepe also do you know that't is the sate of Rust libararies like libblkio or that rustabaga ones in the WRT?

File: utils/qemu/Makefile (hunk 1 above)
Comment (1715173329) [reply utils/qemu/Makefile:380]: Okay so please mention it in your commit message like "disabled new options introduced in this version"

Comment (vooon) [reply utils/qemu/Makefile:380]: @1715173329 done.

**Review by 1715173329 — APPROVED** [review]

File: utils/qemu/Makefile (hunk 3 above)
Comment (1715173329) [reply utils/qemu/Makefile:421]: OpenWrt has rust toolchain, but these packages aren't added yet. You can create new Makefile for them.

Comment (vooon) [reply utils/qemu/Makefile:421]: @1715173329 thanks for the info. I'm not going to use features that requires that libraries, so i'm not the best candidate to maintain those packages.

**Comment by vooon** [comment]
Fiendly ping @BKPepe @yousong @neheb

File: utils/qemu/Makefile [hunk 4]
```diff
@@ -9,10 +9,10 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=qemu
-PKG_VERSION:=8.1.2
+PKG_VERSION:=8.2.0
 PKG_RELEASE:=1
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.xz
-PKG_HASH:=541526a764576eb494d2ff5ec46aeb253e62ea29035d1c23c0a8af4e6cd4f087
+PKG_HASH:=bf00d2fa12010df8b0ade93371def58e632cb32a6bfdc5f5a0ff8e6a1fb1bf32
 PKG_SOURCE_URL:=http://download.qemu.org/
```
Comment (dcousens) [inline utils/qemu/Makefile:16]:
```suggestion
PKG_SOURCE_URL:=https://download.qemu.org/
```
You could probably update this at the same time, it is easily checked by opening https://download.qemu.org/ in your browser

**Pull request merged**